charts/external-dns/Chart.yaml

```diff
@@ -2,10 +2,11 @@ apiVersion: v2
 name: external-dns
 description: ExternalDNS synchronizes exposed Kubernetes Services and Ingresses with DNS providers.
 type: application
-version: 1.9.0
-appVersion: 0.11.0
+version: 1.11.0
+appVersion: 0.12.2
```

Comment:
Does this appVersion refer to the external-dns version? It should be 0.13.1, shouldn't it?

Reply:
Yes, `appVersion` refers to the version of the externalDNS deployed by this help chart. It's always on delay because of the chicken and egg problem: first you need to release and then you need to update the helm chart. 
We don't use the helm chart in anyway, so I think we can ignore that discrepancy.